File: ARCADE/Intro/Level 10_2: findEmailDomain.cs
    using System;
     
    public class Test
    {
    	public static string findEmailDomain(string s) {
        	return s.Substring(s.LastIndexOf('@')+1);
    	}
     
    	public static void Main()
    	{
    		string  address = "prettyandsimple@example.com";
    		Console.WriteLine(findEmailDomain(address));
    	}
    }

File: ARCADE/Intro/Level 10_2: findEmailDomain.pas
program findEmailDomain;

uses Sysutils;

function findEmailDomain(address: string): string;
var i, a : integer;
    rs : string;
begin
    rs := '';
    a := LastDelimiter('@', address);
    for i := a + 1 to length(address) do
        rs := rs + address[i];
    findEmailDomain := rs;
end;

var address : string;
begin
	address := 'prettyandsimple@example.com';
	writeln(findEmailDomain(address));
	
	address := '<>[]:,;@\"!#$%&*+-/=?^_{}| ~.a\"@example.org';
	writeln(findEmailDomain(address));
end.
